### ISSUE TITLE
Can these modules be used for MeshCNN Segmentation?

### ISSUE THREAD

**Issue by EJShim** [ranahanocka/point2mesh#17]
Hello, I found this project while trying to implement MeshCNN.

I can see codes here are based on MeshCNN, but have some new features using pytorch3D.

Do you recommend to use modules defined in this project for implementation of MeshCNN?

If I use ```Mesh```  and ```MeshEncoderDecoder``` module from this project instead of MeshCNN project,

can I expect some performance improvement?

Thank you for your great works

**Comment by ranahanocka** [comment]
Hi @EJShim --

I am not sure they will help for mesh segmentation, they are generally the same functions with some "quality of life improvements" (i.e., the main thing is instead of collapsing to a # of edges we collapse a % of edges, which makes it a bit more modular for different mesh resolutions). The pytorch3d stuff is for the chamfer loss function which is not needed in segmentation.
